src/engraving/dom/arpeggio.cpp

```diff
@@ -425,6 +425,11 @@ bool Arpeggio::crossStaff() const
     return (track() + span() - 1) / VOICES != staffIdx();
 }
 
+staff_idx_t Arpeggio::vStaffIdx() const
+{
+    return chord()->vStaffIdx();
```

Comment:
Perhaps, just to be safe:
```suggestion
    return chord() ? chord()->vStaffIdx() : EngravingItem::vStaffIdx();
```
(keeping in mind an earlier crash caused by something like this...)